pyvista/utilities/reader.py

```diff
@@ -1478,29 +1481,58 @@ def set_active_time_point(self, time_point):  # noqa: D102
         self.set_active_time_value(self.time_values[time_point])
 
 
+class DICOMReader(BaseReader):
+    """DICOM Reader for reading .dcm files.
+
+    Examples
+    --------
+    >>> import pyvista as pv
+    >>> filename = examples.download_dicom_stack(load=False)
+    >>> reader = pv.get_reader(filename)
+    >>> mesh = reader.read()
+    >>> mesh.plot()
+    """
```

Comment:
I think the PyVista standard has a space at the end of multiline docstrings.

```suggestion

    """
```

Reply:
Interesting. Okay, easy enough fix.